include/SFML/Window/Event.hpp

```diff
@@ -100,7 +100,7 @@ class Event
     };
 
     ////////////////////////////////////////////////////////////
-    /// \brief Mouse wheel events parameters (MouseWheelMoved)
+    /// \brief Mouse wheel events parameters (MouseWheelMoved) (deprecated)
```

Comment:
Deprecated could be mentioned with [\deprecated](http://www.stack.nl/~dimitri/doxygen/manual/commands.html#cmddeprecated).